tests/src/test/java/alluxio/client/hadoop/FileSystemRenameIntegrationTest.java

```diff
@@ -236,7 +236,7 @@ public void basicRenameTest7() throws Exception {
     // Due to Hadoop 1 support we stick with the deprecated version. If we drop support for it
     // FSDataOutputStream.hflush will be the new one.
     //#ifdef HADOOP1
-    o.sync();
+//    o.sync();
```

Comment:
@maobaolong  is this change required for this PR?

Reply:
Sorry for this.